File: hello.js
// console.log("Hello 928s");
// console.log('Hello 928s');
// console.log(typeof('Hello 928s'));

// console.log((42).toString());
// console.log(parseInt("42", 10));

// // Binary example (optional)
// console.log(parseInt("101", 10));
// console.log(parseInt("101", 2));

// const student = "Carla";
// let age = 25;

// age += 1;
// console.log(age);

// // student = "Koki"; // NO RE-ASSIGNEMENT FOR CONST
// // console.log(student);

// console.log(`${student} is ${age} years-old next year.`)

const student = "cArLa";

console.log(student[0]);
console.log(student.length);

console.log(student.slice(2)); // from index 2 and after
console.log(student.slice(1, 4)); // from index 1 to index 4 excluded

console.log(student.toUpperCase());
console.log(student.toLowerCase());
// no capitalize in JS!

const students_string = "Song, Mounir, Jan, Soufiane, Josh";
const another_students_string = "Song Mounir Jan Soufiane Josh";

const students = students_string.split(", ");
console.log(students)
console.log(another_students_string.split(" "))

console.log(students.join()); // add commas by default
console.log(students.join(""));

File: hello.rb
# p "Hello 928s"
# p "Hello 928s".class

# p 42.to_s
# p '42'.to_i

# student = "Carla"
# age = 25

# puts "#{student} is #{age} years-old."

# age += 1

# puts "#{student} is #{age} years-old next year."

student = "cArLa"

p student[0]
p student.size
p student.length
# p student.count # ARRAY METHOD

p student.slice(1, 3) #keep 3 chars from index 1

p student.upcase
p student.downcase
p student.capitalize

students_string = "Song, Mounir, Jan, Soufiane, Josh"
another_students_string = "Song Mounir Jan Soufiane Josh"

students = students_string.split(", ")
p students
p another_students_string.split

p students.join
p students.join("~~")